app/router.js

```diff
@@ -14,6 +14,7 @@ Router.map(function() {
   });
   this.route('login');
   this.route('index', { path: '/' });
+  this.route('waiting-list');
```

Comment:
thoughts on nesting this in customers? or stores or artists. i think in the end we don't want this to be a top-level route. (we could also wait on that for a future feature)